ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -312,6 +312,45 @@ def test_tensorflow_Cos(  # NOQA
     )
 
 
+# Cross
+@handle_frontend_test(
+        fn_tree='tensorflow.raw_ops.Cross',
+        dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=3,
+        max_dim_size=3,
+        safety_factor_scale="log",
+         max_value=1e5,
+
```

Comment:
The max_value is not necessary since you added the safety_factor. Please remove the max_value and the empty lines here and in the test function. After that I will merge this!

Reply:
Done removed the spaces and max_value